src/Chocolatey.PowerShell/Helpers/EnvironmentHelper.cs

```diff
@@ -221,37 +231,40 @@ public static void UpdateSession(PSCmdlet cmdlet)
                 scopeList.Add(EnvironmentVariableTarget.User);
             }
 
-            foreach (var scope in scopeList)
+            if (cmdlet.ShouldProcess("current process", "refresh all environment variables"))

```

Comment:
Could we be consistent with these `ShouldProcess` calls on the capitalization used?

See previous comment for suggested style of first letter capitalized.

Reply:
Sorted!